zip-0032.rst

```diff
@@ -549,6 +549,52 @@ each recipient.
 Note that a given account can have a maximum of :math:`2^{88}` payment addresses (unlike Sapling, all Orchard
 diversifiers are valid).
 
+Account discovery
+-----------------
+
+A wallet that imports a master seed from an external source SHOULD attempt to discover accounts in the
+following manner.
```

Comment:
'account' is used and one can infer its meaning from context but it would be nice to have an explicit definition near the top of this document